specification/schemaregistry/SchemaRegistry/examples/2023-07-01/GetSchemaPropertiesByContent.json

```diff
@@ -1,8 +1,10 @@
 {
+  "operationId": "GetSchemaPropertiesByContent",
+  "title": "Get Schema Properties By Content",
   "parameters": {
     "endpoint": "mynamespace.servicebus.windows.net",
     "groupName": "sample-sr-group",
-    "schemaName": "sample-protobuf-schema",
```

Comment:
I dont believe we should have these types of changes in the examples, maybe something got renamed accidentally?

Reply:
I changed the parameter names b/c I get an error when the key and the model are named the same thing, so I re-named the key to just "name". Should I change it back?